service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -275,15 +278,29 @@ void shouldGenerateCorrectFileNameGivenOrderType(GeneratedOrderType type,
     @ParameterizedTest
     @MethodSource("docmosisDataGenerationSource")
     void shouldCreateExpectedMapWhenGivenPopulatedCaseData(GeneratedOrderType orderType,
-                                                           GeneratedOrderSubtype subtype) {
+                                                           GeneratedOrderSubtype subtype) throws IOException {
         LocalDateTime now = time.now();
         CaseData caseData = createPopulatedCaseData(orderType, subtype, now.toLocalDate());
 
         Map<String, Object> expectedMap = createExpectedDocmosisData(orderType, subtype, now);
-        Map<String, Object> templateData = service.getOrderTemplateData(caseData);
+        Map<String, Object> templateData = service.getOrderTemplateData(caseData, SEALED);
 
-        assertThat(templateData).isEqualTo(expectedMap);
+        assertThat(templateData).containsAllEntriesOf(expectedMap);
+        assertThat(templateData).containsKey("courtseal");
+    }
+
+    @ParameterizedTest
+    @MethodSource("docmosisDataGenerationSource")
+    void shouldCreateExpectedMapWhenGivenPopulatedCaseDataInDraft(GeneratedOrderType orderType,
+                                                           GeneratedOrderSubtype subtype) throws IOException {
+        LocalDateTime now = time.now();
+        CaseData caseData = createPopulatedCaseData(orderType, subtype, now.toLocalDate());
+
+        Map<String, Object> expectedMap = createExpectedDocmosisData(orderType, subtype, now);
+        Map<String, Object> templateData = service.getOrderTemplateData(caseData, DRAFT);
 
+        assertThat(templateData).containsAllEntriesOf(expectedMap);
+        assertThat(templateData).containsKey("draftbackground");
```

Comment:
should this assertion not be in expectedMap?

Reply:
see comment above